spec/spec_helper.rb

```diff
@@ -32,6 +32,15 @@ def stub_govuk_repos(repo_names)
     )
 end
 
+ def stub_github_get_pullrequests(repo_name, branch_name)
+  stub_request(:get, "https://api.github.com/repos/alphagov/#{repo_name}/pulls?head=alphagov:#{branch_name}&per_page=100")
+      .to_return(
+        status: 200,
+        headers: { "Content-Type": "application/json" },
+        body: [].to_json,
```

Comment:
Hmmm I'm not sure how your tests are passing. `stub_github_get_pullrequests` stubs the network call in `util.rb`, but you're returning an empty array, which is effectively GitHub API saying "no pull requests with this branch name". So I'd expect [this test](  it "skips repos where the PR already exists" do) to be failing - if it isn't, then it's probably not testing what you think it is 🤔